src/appMain/main.cc

```diff
@@ -125,6 +125,12 @@ int32_t main(int32_t argc, char** argv) {
     profile::Profile::instance()->config_file_name("smartDeviceLink.ini");
   }
 
+  // Unsibscribe once for all threads
+  if ( utils::UnsibscribeFromTermination() ) {
```

Comment:
Not clear for me why we have an error in case of successful unsubscription. Please, remove spaces in `if` statement